### PR TITLE
Update Getting started with Yeoman to csproj

### LINKED ISSUE #2747: aspnetcore/client-side/yeoman.md msbuild conversion
[Building projects with Yeoman](https://docs.microsoft.com/en-us/aspnet/core/client-side/yeoman)

### PR DESCRIPTION
**Pull request by spboyer**
Fixes #2747 

- article content cleaned up
- screen shots updated for macOS
- updated screen shots for project contents 
- updated screen shots for VS 2017

### CODE REVIEW

**Comment by spboyer** [comment]
@Rick-Anderson please review

File: aspnetcore/client-side/yeoman.md [hunk 1]
```diff
@@ -14,21 +14,23 @@ uid: client-side/yeoman
...
 
 ## Install Node.js, npm, and Yeoman
 
-To get started with Yeoman install [Node.js](https://nodejs.org/en/). The installer includes [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/).
+### Prerequisites
+
+Node.js and npm are required for Yeoman. Download from [Node.js](https://nodejs.org/en/), the installer includes [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/).
 
-Follow the instructions on [http://yeoman.io/learning/](http://yeoman.io/learning/) to install [yo](https://github.com/yeoman/yo), bower, grunt, and gulp.
+To install the Yeoman generator run the following command. Bower is also installed for accessing UI components like stylesheets.
```
Comment (Rick-Anderson) [inline aspnetcore/client-side/yeoman.md]: ~~To install the Yeoman generator run the following command. Bower is also installed for accessing UI components like stylesheets.~~

File: aspnetcore/client-side/yeoman.md [hunk 2]
```diff
@@ -14,21 +14,23 @@ uid: client-side/yeoman
...
+To install Yeoman run the following command:
 
 ```console
 npm install -g yo bower
 ```
 
-Note: If you get the error `npm ERR! Please try running this command again as root/Administrator.` on Mac OS, run the following command using [sudo](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man8/sudo.8.html): `sudo npm install -g yo bower`
+>[!Note]
```
Comment (Rick-Anderson) [inline aspnetcore/client-side/yeoman.md]: Above the note add 

 Bower is also installed for accessing UI components such as style-sheets.

File: aspnetcore/client-side/yeoman.md [hunk 3]
```diff
@@ -14,21 +14,23 @@ uid: client-side/yeoman
...
-
-[Yeoman](http://yeoman.io/) generates complete projects for a given set of client tools. Yeoman is an open-source tool that works like a Visual Studio project template. The Yeoman command-line tool [yo](https://github.com/yeoman/yo) works alongside a Yeoman generator. Generators define the technologies that go into a project.
+[Yeoman](http://yeoman.io/) is a project scaffolding system for creating many kinds of applications. Similar to Visual Studio, the Yeoman generator for ASP.NET Core contains a variety of project templates for starting a new web, MVC, or console application.
 
 ## Install Node.js, npm, and Yeoman
 
-To get started with Yeoman install [Node.js](https://nodejs.org/en/). The installer includes [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/).
+### Prerequisites
+
+Node.js and npm are required for Yeoman. Download from [Node.js](https://nodejs.org/en/), the installer includes [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/).
```
Comment (Rick-Anderson) [inline aspnetcore/client-side/yeoman.md]: Node.js and npm are required for Yeoman. Download from [Node.js](https://nodejs.org/en/).  The installer includes

File: aspnetcore/client-side/yeoman.md [hunk 4]
```diff
@@ -14,21 +14,23 @@ uid: client-side/yeoman
 ---
 # Building projects with Yeoman
 
-By [Steve Smith](http://ardalis.com), [Scott Addie](https://scottaddie.com), [Rick Anderson](https://twitter.com/RickAndMSFT), [Noel Rice](http://blog.falafel.com/author/noel-rice/), and [Shayne Boyer](https://twitter.com/spboyer)
-
-[Yeoman](http://yeoman.io/) generates complete projects for a given set of client tools. Yeoman is an open-source tool that works like a Visual Studio project template. The Yeoman command-line tool [yo](https://github.com/yeoman/yo) works alongside a Yeoman generator. Generators define the technologies that go into a project.
+[Yeoman](http://yeoman.io/) is a project scaffolding system for creating many kinds of applications. Similar to Visual Studio, the Yeoman generator for ASP.NET Core contains a variety of project templates for starting a new web, MVC, or console application.
```
Comment (spboyer) [inline aspnetcore/client-side/yeoman.md]: Good point updated this.

File: aspnetcore/client-side/yeoman.md [hunk 5]
```diff
@@ -14,21 +14,23 @@ uid: client-side/yeoman
...
 
 ## Install Node.js, npm, and Yeoman
 
-To get started with Yeoman install [Node.js](https://nodejs.org/en/). The installer includes [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/).
+### Prerequisites
```
Comment (spboyer) [inline aspnetcore/client-side/yeoman.md]: Should I change this to a NOTE? @RickAndMSFT

File: aspnetcore/client-side/yeoman.md (hunk 3 above)
Comment (spboyer) [inline aspnetcore/client-side/yeoman.md]: Made the change.  Didn't like the original sentence was trying to make it read better.

Comment (spboyer) [inline aspnetcore/client-side/yeoman.md]: see comment below..

File: aspnetcore/client-side/yeoman.md (hunk 1 above)
Comment (spboyer) [inline aspnetcore/client-side/yeoman.md]: oops, meant to remove repetitive sentence. looking to improve original.

**Pull request merged**